ComputeExamples.md

```diff
@@ -0,0 +1,676 @@
+  const encryptedResponse = await providerResponse
+  const validateResult = await aquarius.validate(ddo)
+```
+<!--
+  assert(validateResult.valid, 'Could not validate metadata')
+-->
+```Typescript
+  await nft.setMetadata(
+    nftAddress,
+    owner,
+    0,
+    providerUrl,
+    '',
+    '0x2',
+    encryptedResponse,
+    validateResult.hash
+  )
+  return ddo.id
+}
+```
+
+Add a `handleOrder()`function.
+```Typescript
+async function handleOrder(
+  order: ProviderComputeInitialize,
+  datatokenAddress: string,
+  payerAccount: string,
+  consumerAccount: string,
+  serviceIndex: number,
+  consumeMarkerFee?: ConsumeMarketFee
+) {
+  const datatoken = new Datatoken(web3)
+  const config = await getTestConfig(web3)
+  /* We do have 3 possible situations:
+     - have validOrder and no providerFees -> then order is valid, providerFees are valid, just use it in startCompute
+     - have validOrder and providerFees -> then order is valid but providerFees are not valid, we need to call reuseOrder and pay only providerFees
+     - no validOrder -> we need to call startOrder, to pay 1 DT & providerFees
+  */
+  if (order.providerFee && order.providerFee.providerFeeAmount) {
+    await approveWei(
+      web3,
+      config,
+      payerAccount,
+      order.providerFee.providerFeeToken,
+      datatokenAddress,
+      order.providerFee.providerFeeAmount
+    )
+  }
+  if (order.validOrder) {
+    if (!order.providerFee) return order.validOrder
+    const tx = await datatoken.reuseOrder(
+      datatokenAddress,
+      payerAccount,
+      order.validOrder,
+      order.providerFee
+    )
+    return tx.transactionHash
+  }
+  const tx = await datatoken.startOrder(
+    datatokenAddress,
+    payerAccount,
+    consumerAccount,
+    serviceIndex,
+    order.providerFee,
+    consumeMarkerFee
+  )
+  return tx.transactionHash
+}
+```
+
+<!--
+describe('Compute to date example tests
+-->
+
+We load the configuration:
+```Typescript
+  
+    config = await getTestConfig(web3)
+    addresses = getAddresses()
+    aquarius = new Aquarius(config.metadataCacheUri)
+    providerUrl = config.providerUri
+    datatoken = new Datatoken(web3)
+```
+As we go along it's a good idea to console log the values so that you check they are right
+```Typescript
+    console.log(`Aquarius URL: ${config.metadataCacheUri}`)
+    console.log(`Provider URL: ${providerUrl}`)
+    console.log(`Deployed contracts address: ${addresses}`)
+  
+```
+
+## 5. Initialize accounts
+  ### 5.1 Initialize accounts
+```Typescript
+    const accounts = await web3.eth.getAccounts()
+    publisherAccount = accounts[0]
+    consumerAccount = accounts[1]
+```
+Again, lets console log the values so that we can check that they have been saved properly
+```Typescript
+    console.log(`Publisher account address: ${publisherAccount}`)
+    console.log(`Consumer account address: ${consumerAccount}`)
+  
+```
+
+  ### 5.2 Mint OCEAN to publisher account
```

Comment:
we tell people to call `.mint()` on the OCEAN token contract? This will never work

Reply:
Our integration tests are using barge, since this readme is generated based on a compute integration test we kind off need this to be there. Going to mention tho over here that this step is needed just if the users are going to run the script against barge otherwise they can skip this step.